score(Collector) is not abstract yet, and we are already in Lucene 3.0.

--- package.html	(revision 980793)
+++ package.html	(working copy)
@@ -345,8 +345,8 @@
     <p>The
         <a href="Scorer.html">Scorer</a>
         abstract class provides common scoring functionality for all Scorer implementations and
-        is the heart of the Lucene scoring process. The Scorer defines the following abstract (they are not
-        yet abstract, but will be in Lucene 3.0 and should be considered as such now) methods which
+        is the heart of the Lucene scoring process. The Scorer defines the following abstract (some of them are not
+        yet abstract, but will be in future versions and should be considered as such now) methods which
         must be implemented (some of them inherited from <a href="DocIdSetIterator.html">DocIdSetIterator</a> ):
         <ol>
             <li>
